.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+on: [pull_request]
+
+jobs:
+  beefore:
+    name: Pre-test checks
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        task: ['pycodestyle']
```

Comment:
Yeah - I should clean up that naming problem. At one point in the (distant) past, it *did* use pycodestsyle; but in moving to flake, I didn't update the name.

The build command is a matrix (of one) because, in theory, other pre-test checks can be added. There's no (as far as I can make out) downside to having a matrix of one, and if we ever get around to adding other checks, the config diff will be a lot cleaner.